internal/docker/docker.go

```diff
@@ -199,8 +199,7 @@ func InspectContainer(name string) (*types.ContainerJSON, error) {
 	ctx := context.Background()
 
 	labelFilters := filters.NewArgs()
-	labelFilters.Add("label", "service.owner=co.elastic.observability")
-	labelFilters.Add("label", "service.container.name="+name)
+	labelFilters.Add("label", "com.docker.compose.service="+name)
```

Comment:
While working on the docker copy stuff, I noticed this was wrong from the beginning. Instead, we should use:
```suggestion
	labelFilters.Add("name", name)
```